tests/tck/conftest.py

```diff
@@ -980,3 +980,19 @@ def switch_to_new_session(conn_pool, user, password, class_fixture_variables, ex
     sess = conn_pool.get_session(user, password)
     class_fixture_variables["sessions"].append(sess)
     exec_ctx["current_session"] = sess
+
+@when(parse('login with user "{user}"'))
+def login_without_password(conn_pool, user):
+    sess = None
+    try:
+        sess = conn_pool.get_session(user, '')
+    except Exception as e:
+        assert e
+
+@when(parse('login with user "{user}" and "{password}"'))
```

Comment:
plz given the password keyword to indicate what the following string is.

```
login with user "{user}" and password "{password}"
```

Reply:
ok